client/example6/bundle/config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"id":"defaultProcess",
+	"name": "process.name",
+	"version":"2",
+	"states": {
+		"messageState": {
+		  "name": "Message",
+		  "description": "Message",
+		  "color": "#CC9900",
+		  "userCard" : {
+			"template" : "usercard_message",
+			"endDateVisible" : false,
+			"lttdVisible" : false,
+			"expirationDateVisible" : false
+		  },
+		  "templateName": "message",
+		  "styles": [],
+		  "acknowledgmentAllowed": "Always",
+		  "showAcknowledgmentFooter" : "OnlyForEmittingEntity",
+		  "type" : "INPROGRESS",
+		  "consideredAcknowledgedForUserWhen" : "AllEntitiesOfUserHaveAcknowledged"
```

Comment:
Remove unnecessary params for the example : styles , acknowledgmentAllowed , showAcknowledgmentFooter , type , consideredAcknowledgedForUserWhen , color